locales/vi/vi.json

```
@@ -272,7 +272,7 @@
     "If you already have an account, you may accept this invitation by clicking the button below:": "Nếu bạn đã có tài khoản, bạn có thể chấp nhận lời mời này bằng cách bấm vào nút bên dưới:",
     "If you did not create an account, no further action is required.": "Nếu bạn không đăng ký tài khoản này, bạn không cần thực hiện thêm hành động nào.",
     "If you did not expect to receive an invitation to this team, you may discard this email.": "Nếu bạn không muốn chấp nhận lời mời, bạn có thể bỏ qua email này.",
-    "If you did not receive the email": "If you did not receive the email",
+    "If you did not receive the email": "Nếu bạn không nhận được email.",
```

```suggestion
    "If you did not receive the email": "Nếu bạn không nhận được email",
```

The full stop at the end of the sentence is redundant.